Core/Keyed/Dialogs_Various.xml

```diff
@@ -527,7 +527,7 @@
   <!-- EN: {0} / hour -->
   <PerHour>{0} / hodinu</PerHour>
   <!-- EN: {0} XP / hour -->
-  <XpPerHour>TODO</XpPerHour>
+  <XpPerHour>{0} zkušeností / hodinu</XpPerHour>
```

Comment:
nestačí zk. ? vleze se?

Reply:
vleze se to, je to v tom tooltipu a tam se stejně kde co zalamuje
může tam být i třeba více dovedností najednou